### ISSUE TITLE
Incorrect default value of Session attributes in documentation

### ISSUE THREAD

**Issue by lmazuel** [psf/requests#4213]
See for instance `trust_env`, that is `True` by default, but `None` in the documentation:
http://docs.python-requests.org/en/latest/api/#requests.Session.trust_env

In that case, True/None are opposite, and I understood my own issue only after looking at the code :/

Same for all attributes actually, that are all `None` in doc.

I totally get that it's complicated for sphinx to generate instance attribute (since the value is assigned from inside `__init__`), but at least would be nice to have like `max_redirects` the default value in the docstring:

![image](https://user-images.githubusercontent.com/1050156/28797272-99d8b8c6-75f5-11e7-81fa-59e1298f4e40.png)

I could probably try to PR some of these strings if it helps and if you confirm the "This defaults to XXXX" pattern in the docstring.

Thanks! :-)

**Comment by Lukasa** [comment]
Hrm, Sphinx autodoc should be getting those rights they're very simple. Rather than change the text, which will end up out of sync with the code, it may be worth digging in to why Sphinx is failing to detect this.

**Comment by sethmlarson** [comment]
Defaults are no longer displayed in the documentation, closing this issue.
